app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -170,17 +170,12 @@ def raw_connect(api_key, pcloud_guid)
       end
 
       require "ibm_cloud_resource_controller"
-      api_client = IbmCloudResourceController::ApiClient.new
-      api_client.config.api_key        = {"Authorization" => token.access_token}
-      api_client.config.api_key_prefix = {"Authorization" => token.token_type}
-      api_client.config.access_token   = {"Authorization" => token.access_token}
-      api_client.config.logger         = $ibm_cloud_log
-
-      resource_instances_api = IbmCloudResourceController::ResourceInstancesApi.new(api_client)
+      authenticator = IbmCloudResourceController::Authenticators::BearerTokenAuthenticator.new(:bearer_token => token.access_token)
+      resource_controller_api = IbmCloudResourceController::ResourceControllerV2.new(:authenticator => authenticator)
 
       begin
-        power_iaas_service = resource_instances_api.get_resource_instance(pcloud_guid)
-      rescue IbmCloudResourceController::ApiError => err
+        power_iaas_service = resource_controller_api.get_resource_instance(:id => pcloud_guid).result
+      rescue IbmCloudResourceController::ApiException => err
```

Comment:
The `ApiException` is structured differently than `ApiError`. See the base class here: https://github.com/IBM/ruby-sdk-core/blob/main/lib/ibm_cloud_sdk_core/api_exception.rb#L7

As a result, to get the "error_message" when the GUID is invalid we need to now use the `err.error` instance variable. We could also print the entire exception since a nice `to_s` method is provided... but I prefer short and to the point.

Here's the result from using `err.error`:
![image](https://user-images.githubusercontent.com/1637291/114917285-673f2900-9deb-11eb-8c6c-4fc6d90efe67.png)